ImageSegmentation/ExtractContourWithSnakes.cxx

```diff
@@ -47,7 +47,7 @@ int main( int argc, char* argv[] )
     std::cout << "Usage " << argv[0]
               << " points alpha beta gamma sigma iterations [image]"
               << std::endl;
-    return EXIT_SUCCESS;;
+    return EXIT_FAILURE;;
```

Comment:
Side-note: there is an extra semi-colon here

Reply:
Done in 3764fad.